### PR TITLE
Removed use of aria-expanded.

### LINKED ISSUE #79: iron-collapse should *not* set aria-expanded
### Description
iron-collapse incorrectly sets the aria-expanded attribute.

According to W3C aria-expanded should be set on the ["triggering link or button"](https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element#Description).

iron-collapse is not the trigger link or button but rather the collapsible content itself.

Developers should set aria-expanded on the button that *triggers* the opening/closing of iron-collapse and it should *not* be set on iron-collapse itself.

### Expected outcome

When navigating into an iron-collapse region the screen reader should not indicate that the region is expanded.

### Actual outcome

When navigating into an expanded iron-collapse region the screen reader announces 'Expanded'. 

### Browsers Affected
I've only tested this on Chrome, but I suspect it applies everywhere a screen reader is used.
- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

### PR DESCRIPTION
**Pull request by TomHennen**
Fixes #79 by not setting aria-expanded.

### CODE REVIEW

File: test/a11y.html [hunk 1]
```diff
@@ -47,13 +47,11 @@
 
         test('aria attributes', function() {
           assert.equal(collapse.getAttribute('role'), 'group');
-          assert.equal(collapse.getAttribute('aria-expanded'), 'false');
```
Comment (merlinnot) [inline test/a11y.html]: You can actually assert that it's undefined and add a comment explaining why for future reference.

Comment (TomHennen) [reply test/a11y.html]: Done.

**Comment by valdrinkoshi** [comment]
Sweet, thanks! Mind updating also the example in the README.md?

----
[...] show/hide the content. The `aria-expanded` attribute should only be set on the button that
controls the collapsable area, not on the area itself. See https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element#Description

```html
<button id="button" on-click="toggle">toggle collapse</button>

<iron-collapse id="collapse">
  <div>Content goes here...</div>
</iron-collapse>

...

toggle: function() {
  this.$.collapse.toggle();
  this.$.button.setAttribute('aria-expanded', this.$.collapse.opened);
}
```

File: README.md [hunk 2]
```diff
@@ -6,7 +6,10 @@ _[Demo and API docs](https://elements.polymer-project.org/elements/iron-collapse
...
 
 `iron-collapse` creates a collapsible block of content.  By default, the content
-will be collapsed.  Use `opened` or `toggle()` to show/hide the content.
+will be collapsed.  Use `opened` or `toggle()` to show/hide the content. The
+aria-expanded attribute should only be set on the button that controls the
+collapsable area, not on the area itself. See
+https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element#Description
 
 ```html
 <button on-click="toggle">toggle collapse</button>
```
Comment (valdrinkoshi) [inline README.md]: We're missing the `id="button"` here (in case someone copy-pastes the code)

File: test/a11y.html [hunk 3]
```diff
@@ -47,13 +47,15 @@
...
           assert.equal(collapse.getAttribute('aria-hidden'), 'true');
+          // aria-expanded should only be set on the button that
+          // controls the collapsable area, not on the area itself.
+          // See https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element#Description
+          assert.isNotOk(collapse.getAttribute('aria-expanded'));
```
Comment (valdrinkoshi) [inline test/a11y.html]: nit: I'd use `assert.isFalse(collapse.hasAttribute('aria-expanded'))` here, to also verify that we don't set the attribute itself.

**Comment by valdrinkoshi** [comment]
I'll merge this and address my comment in a separate PR 👌 
Thanks @TomHennen for the contribution!

**Comment by TomHennen** [comment]
Great, thanks for taking a look so promptly!

**Pull request merged**